Mk/subdir.mk

```diff
@@ -43,7 +43,7 @@ else
 INSTALL_ARGS	+= lib_LTLIBRARIES= pkginclude_HEADERS= pkgconfig_DATA=
   ifneq (${self_sub},modules)
    ifeq (${self_sub},modules_afsocket)
-INSTALL_ARGS	+= module_LTLIBRARIES="modules/afsocket/libafsocket-notls.la modules/afsocket/libafsocket-tls.la"
```

Comment:
You should use it: 
INSTALL_ARGS	+= module_LTLIBRARIES="modules/afsocket/libafsocket.la